auxiliary_files/CMakeLists.txt

```diff
@@ -0,0 +1,86 @@
+# Copyright 2022 The Mumble Developers. All rights reserved.
+# Use of this source code is governed by a BSD-style license
+# that can be found in the LICENSE file at the root of the
+# Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+include(delayed_configure_files)
+
+if(NOT BUILD_RELEASE_DATE)
+	# If BUILD_RELEASE_DATE has not been set, default to time of build
+	string(TIMESTAMP BUILD_RELEASE_DATE "%Y-%m-%d")
+endif()
+
+if(client)
+	if(UNIX AND NOT APPLE)
+		delayed_configure_files(
+			TARGET mumble
+			FILES
+			"${CMAKE_CURRENT_SOURCE_DIR}/config_files/info.mumble.Mumble.appdata.xml.in=${CMAKE_CURRENT_BINARY_DIR}/info.mumble.Mumble.appdata.xml"
+			"${CMAKE_CURRENT_SOURCE_DIR}/config_files/info.mumble.Mumble.desktop.in=${CMAKE_CURRENT_BINARY_DIR}/info.mumble.Mumble.desktop"
+			PASSTHROUGH_VARIABLES
+				MUMBLE_BUILD_YEAR
+				CMAKE_PROJECT_VERSION
+				BUILD_RELEASE_DATE
+				CMAKE_PROJECT_HOMEPAGE_URL
+				CMAKE_PROJECT_DESCRIPTION
+			VARIABLES
+				"MUMBLE_CLIENT_BINARY_NAME=$<TARGET_FILE_NAME:mumble>"
+			PYTHON_INTERPRETER "${PYTHON_INTERPRETER}"
+			@ONLY
+		)
+
+		install(FILES "${CMAKE_BINARY_DIR}/info.mumble.Mumble.appdata.xml" DESTINATION "${CMAKE_INSTALL_DATAROOTDIR}/metainfo")
+		install(FILES "${CMAKE_BINARY_DIR}/scripts/info.mumble.Mumble.desktop" DESTINATION "${CMAKE_INSTALL_DATAROOTDIR}/applications")
+
+		# Install Mumble client man files
+		install(FILES "man_files/mumble.1" DESTINATION "${MUMBLE_INSTALL_MANDIR}" COMPONENT doc)
+
+	endif()
+endif()
+
+if(server)
+	file(COPY "mumble-server.ini" DESTINATION ${CMAKE_BINARY_DIR})
+
+	if(UNIX AND NOT APPLE)
+		delayed_configure_files(
+			TARGET mumble-server
+			FILES
+				"${CMAKE_CURRENT_SOURCE_DIR}/config_files/mumble-server.logrotate.in=${CMAKE_CURRENT_BINARY_DIR}/mumble-server.logrotate"
+				"${CMAKE_CURRENT_SOURCE_DIR}/config_files/mumble-server.service.in=${CMAKE_CURRENT_BINARY_DIR}/mumble-server.service"
+				"${CMAKE_CURRENT_SOURCE_DIR}/run_scripts/mumble-server-user-wrapper.in=${CMAKE_CURRENT_BINARY_DIR}/mumble-server-user-wrapper"
```

Comment:
The install dir for the script can again be derived with the help of `${CMAKE_INSTALL_PREFIX}/bin/`